2160.minimum-sum-of-four-digit-number-after-splitting-digits.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/minimum-sum-of-four-digit-number-after-splitting-digits/
+ */
+
+/**
+ * @param {number} num
+ * @return {number}
+ */
+var minimumSum = function(num) {
+    const number = num.toString().split("").sort();
```

Comment:
```suggestion
    const numbers = num.toString().split("").sort();
```
I think "numbers" seems better.

Reply:
I corrected it